### PR TITLE
GCC 11.3

### PR DESCRIPTION
**Pull request by nmschulte**
for https://github.com/rusefi/rusefi/pull/4680#issuecomment-1306623569

### CODE REVIEW

**Comment by mck1117** [comment]
You'll need the firmware yaml too

File: .github/workflows/build-firmware.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on: [push, pull_request, workflow_dispatch]
 jobs:
   build-firmware:
```
Comment (nmschulte) [inline .github/workflows/build-firmware.yaml:6]: Does `build-primary-bundle` job receive special treatment, or should it also be updated?

**Comment by nmschulte** [comment]
Apologies for all of the CI build hog/spam ... I ~~know~~ knew not what I ~~do~~ did when I created the PRs.

**Comment by rusefillc** [comment]
@nmschulte thank you for your help! where is your forum build thread by the way :)

**Comment by nmschulte** [comment]
> HW CI looks unhappy about PATH?

I saw that, and I'm unsure what the difference is.  Possibly CI depends on the correct job running prior, and I'd not [yet] updated the `.github/workflows/build-firmware.yaml` file by that time?  Unsure; the diff looks good to me.

**Comment by nmschulte** [comment]
> @nmschulte thank you for your help! where is your forum build thread by the way :)

🙇 -- I am actually not using RusEFI; in the middle of restoring a vehicle I'll be putting a Toyota 2JZ into and hope to use it then.  I need RusEFI hardware yet, might try targeting RusEFI to RISC-V and have pipe dreams about porting to a different RTOS though that's just because I know nothing about ChibiOS.

I tried to join the Slack but the heroku invite app seems borked; I guess I could join the forum if that's where the discussion happens.

File: .github/workflows/build-firmware.yaml [hunk 2]
```diff
@@ -318,10 +318,10 @@ jobs:
       if: ${{ env.skip != 'true' }}
       env:
         ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
-      run: | 
-        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.2-2022.02/binrel/gcc-arm-11.2-2022.02-x86_64-arm-none-eabi.tar.xz' -O compiler.tar.xz
+      run: |
+        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz' -O compiler.tar.xz
```
Comment (nmschulte) [inline .github/workflows/build-firmware.yaml:322]: `--progress=dot:giga` will reduce the log size:

```suggestion
        wget 'https://developer.arm.com/-/media/Files/downloads/gnu/11.3.rel1/binrel/arm-gnu-toolchain-11.3.rel1-x86_64-arm-none-eabi.tar.xz' --progress=dot:giga -O compiler.tar.xz
```

https://www.gnu.org/software/wget/manual/html_node/Download-Options.html#Download-Options

**Comment by mck1117** [comment]
> pipe dreams about porting to a different RTOS though that's just because I know nothing about ChibiOS

This is a really bad reason 😜

**Comment by mck1117** [comment]
I've manually downloaded the compiler on the hardware CI machines, if you push an update it should re-run and pass now

**Comment by rusefillc** [comment]
@mi-hol @nmschulte see https://github.com/rusefi/rusefi_documentation/issues/246

**Pull request merged**